example/example.go

```diff
@@ -0,0 +1,62 @@
+// Package main contains an example use of this library; it
+// prints the current Fedora CoreOS EC2(AWS) x86_64 AMI in the
+// us-east-2 region.
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/coreos/stream-metadata-go/fedoracoreos"
+	"github.com/coreos/stream-metadata-go/stream"
+)
+
+const (
+	targetArch = "x86_64"
+	region     = "us-east-2"
+)
+
+func run() error {
+	streamurl := fedoracoreos.GetStreamURL(fedoracoreos.StreamStable)
+	resp, err := http.Get(streamurl.String())
+	if err != nil {
+		return err
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	resp.Body.Close()
+	if err != nil {
+		return err
+	}
+
+	var fcosstable stream.Stream
+	err = json.Unmarshal(body, &fcosstable)
+	if err != nil {
+		return err
+	}
+	var arch stream.Arch
+	var ok bool
+	if arch, ok = fcosstable.Architectures[targetArch]; !ok {
```

Comment:
This is okay, but a more succinct way to write it is

```suggestion
	arch, ok := fcosstable.Architectures[targetArch]
	if !ok {
```